extensions/amp-twitter/validator-amp-twitter.protoascii

```diff
@@ -20,6 +20,7 @@ tags: {  # amp-twitter
   extension_spec: {
     name: "amp-twitter"
     version: "0.1"
+    version: "1.0"
     version: "latest"
     requires_usage: EXEMPTED
     deprecated_allow_duplicates: true
```

Comment:
When adding validations for 1.0 where 0.1 had these looser specifications (usage not required, duplicates allowed), please use `satisfies` and `excludes`. #33478 is an exemplary PR to reference which does exactly this.